graph2tac/loader/predict_server.py

```diff
@@ -319,15 +342,18 @@ def check_alignment(msg):
             unalignedDefinitions = unaligned_definitions,
         )
 
-def prediction_loop(predict_server, capnp_socket, record_file):
-    if predict_server.config.with_meter:
-        capnp_socket = tqdm.tqdm(capnp_socket)
+def prediction_loop(predict_server: PredictServer, capnp_socket: socket.socket, record_file: Optional[IO]):
+    
     message_generator = capnp_message_generator(capnp_socket, record_file)
+    if predict_server.config.with_meter:
+        message_generator_iterator = tqdm.tqdm(message_generator)
+    else:
+        message_generator_iterator = message_generator
```

Comment:
Naming it `message_iterator` instead of `message_generator_iterator` is an improvement but I still don't see the point of your change in the first place -- I find it quite redundant, it is just a little decoration of the generator, and there is no meaning in using the undecorated version afterwards